crates/viewer/re_data_ui/src/blob.rs

```diff
@@ -264,7 +267,19 @@ fn show_video_blob_info(
                 }
             });
         }
+        Err(VideoLoadError::MediaTypeIsNotAVideo(_)) => {
+            // Don't show an error if this wasn't a video in the first place.
+            // Unfortunately we can't easily detect here if the Blob was _supposed_ to be a video, for that we'd need tagged components!
+            // (User may have confidently logged a non-video format as Video, we should tell them that!)
+        }
         Err(err) => {
+            if media_type.is_none() && matches!(err, VideoLoadError::UnrecognizedVideoFormat) {
```

Comment:
My only thought here is given that the only reason we are plumbing through media_type is to check this error condition, I wonder if this should just be bundled with the error-type?
```
    #[error("Video file has unsupported format")]
    UnrecognizedVideoFormat(Option<String>),
```

Reply:
good idea